.github/workflows/test.yml

```diff
@@ -52,20 +50,42 @@ jobs:
         runningBranch: kind
         bcc_version: 0.25.0
         kubectl_version: 1.27.2
-        local_dev_cluster_version: v0.0.0
+        local_dev_cluster_version: main
```

Comment:
let's stay with tag or sha

Reply:
no sha, branch or tag.
here we use the main branch to keep the latest dev version.

Reply:
I totally agree with you @SamYuan1990 on this. I gave this a thought and in some situations where there are some enhancements/major changes done to local_dev_cluster(which are experimental) and we don't want to support that change everywhere else then in that situation sha or tagging might help us to be on stable side. I might be wrong with this assumption. Wdyt?

Reply:
I suppose using main here is good enough for now. @rootfs wdyt?